openedx/core/djangoapps/programs/utils.py

```diff
@@ -340,13 +340,15 @@ def supplement_program_data(program_data, user):
             end_date = course_overview.end or datetime.datetime.max.replace(tzinfo=pytz.UTC)
             run_mode['start_date'] = start_date.strftime(human_friendly_format)
             run_mode['end_date'] = end_date.strftime(human_friendly_format)
+            run_mode['course_ended'] = end_date < timezone.now()
```

Comment:
Nitpick: to make for more readable logic, let's stay consistent with the key names we use for Booleans. If I was making these changes, I'd use `is_course_open` here (flipping the inequality as appropriate), much like we use `is_enrollment_open` below.